cmd/gopass-jsonapi/internal/jsonapi/api_test.go

```diff
@@ -177,16 +177,16 @@ sub:
 
 	runRespondMessage(t,
 		`{"type":"getData","entry":"foo"}`,
-		`{"hallo":"welt","password":"20"}`,
```

Comment:
I'm having a hard time making changes to the jsonapi code because it differs much, so I've filed #1673 to move it to a different repo. For now I tend to ignore this change.